src/NHSD.BuyingCatalogue.Ordering.Api/Controllers/CatalogueSolutionsController.cs

```diff
@@ -77,31 +85,60 @@ public async Task<ActionResult> UpdateAsync(string orderId)
         [Route("{orderItemId}")]
         public ActionResult<CreateOrderItemModel> GetOrderItem(string orderId, string orderItemId)
         {
-            var orderItemKey = GetOrderItemKey(orderId,orderItemId);
+            var orderItemKey = GetOrderItemKey(orderId, orderItemId);
 
             if (CatalogueSolutionOrderItems.ContainsKey(orderItemKey))
             {
                 return CatalogueSolutionOrderItems[orderItemKey];
             }
-            else
+
+            return new CreateOrderItemModel
             {
-                return new CreateOrderItemModel
+                ServiceRecipient = new ServiceRecipientModel
                 {
-                    ServiceRecipient = new ServiceRecipientModel
-                    {
-                        OdsCode = "OX3"
-                    },
-                    SolutionId = orderItemId,
-                    CurrencyCode = "GBP",
-                    DeliveryDate = "2020-04-27",
-                    EstimationPeriod = "month",
-                    ItemUnitModel = new ItemUnitModel { Description = "per consultation", Name = "consultation" },
-                    Price = 0.1m,
-                    ProvisioningType = "OnDemand",
-                    Quantity = 3,
-                    Type = "flat"
-                };
+                    OdsCode = "OX3"
+                },
+                CatalogueSolutionId = orderItemId,
+                CurrencyCode = "GBP",
+                DeliveryDate = DateTime.UtcNow,
+                EstimationPeriod = "month",
+                ItemUnitModel = new ItemUnitModel { Description = "per consultation", Name = "consultation" },
```

Comment:
This should be ItemUnit not itemUnitModel for the property name, I've fixed it In my validation change, up to you whether to make the change here too

Reply:
Yeah I should really have left that endpoint alone. I will leave it because it can be corrected in the implementation of the Get OrderItem endpoint. Good spot though 👍